docs/api/field_type_testing.md

```diff
@@ -1,30 +1,36 @@
+The integration tests with the Persistence SPI can be found in `eZ\Publish\SPI\Tests\FieldType`. In order to implement a test for your custom Field Type, you need to extend the common base class `eZ\Publish\SPI\Tests\FieldType\BaseIntegrationTest` and implement its abstract methods.
 
-The gateway configuration for basic field types are located in [eZ/Publish/Core/settings/storage_engines/legacy/external_storage_gateways.yml](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/settings/storage_engines/legacy/external_storage_gateways.yml).
+!!! tip
+
+    You can use `KeywordIntegrationTest`, `UrlIntegrationTest` and `UserIntegrationTest` as reference.
+
+    Gateway configuration for built-in Field Types is located in [`eZ/Publish/Core/settings/storage_engines/legacy/external_storage_gateways.yml`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/settings/storage_engines/legacy/external_storage_gateways.yml).
 
 ## Public API
 
-On a second level, the interaction between an implementation of the Public API (aka the Business Layer) and the Field Type is tested. Again, there is a common base class as the infrastructural basis for such tests, which resides in `eZ\Publish\API\Repository\Tests\FieldType\BaseIntegrationTest`.
+On the second level, the interaction between an implementation of the Public API and the Field Type is tested.
+Again, there is a common base class as the infrastructural basis for such tests,
+which resides in `eZ\Publish\API\Repository\Tests\FieldType\BaseIntegrationTest`.
 
 !!! note
 
-    Note that the In-Memory stubs for the Public API integration test suite, do not perform actual Field Type calls, but mainly emulate the behavior of a Field Type for simplicity reasons. 
-    
-    If your Field Type needs to convert data between `storeFieldData()` and `getFieldData()`, you need to implement a `eZ\Publish\API\Repository\Tests\Stubs\PseudoExternalStorage` in addition, which performs this task. Running the tests against the Business Layer implementation of the Public API is not affected by this.
\ No newline at end of file
+    Note that the In-Memory stubs for the Public API integration test suite do not perform actual Field Type calls, but mainly emulate the behavior of a Field Type for the sake of simplicity.
+
+    If your Field Type needs to convert data between `storeFieldData()` and `getFieldData()`, you need to additionally implement an `eZ\Publish\API\Repository\Tests\Stubs\PseudoExternalStorage`, which performs this task. Running the tests against the business layer implementation of the Public API is not affected by this.
```

Comment:
@adamwojs all remaining paths seem to be correct, with the exception of `eZ\Publish\API\Repository\Tests\Stubs\PseudoExternalStorage`, which does not seem to exist at all. Do you happen to have an idea what replaced it?